### DISCUSSION TITLE
How would i make it so when i get 50 rings and pass a checkpoint a portal appers above it

### DISCUSSION THREAD

**Discussion by Raptorq1** [Techokami/SonicWorldsNext#95] [Q&A]
How would i make it so when i get 50 rings and pass a checkpoint a portal appers above it

**Comment by DimensionWarped** [comment]
>when I get 50 rings

Add a check to your checkpoint activation code for this exact condition...

>a portal appears above it...

And if that check is passed, spawn your new portal object. (Whether that's just stars on the checkpoint or whatever it might be)

And no, don't expect anyone to code something like that for you, this is very basic stuff that you need to learn to do on your own (and has plenty of adjacent examples already).
